test/replication-luatest/gh_6860_election_off_demote_test.lua

```diff
@@ -0,0 +1,196 @@
+local t = require('luatest')
+local server = require('luatest.server')
+local replicaset = require('luatest.replica_set')
+
+local g = t.group('gh-6860')
+
+local function replicaset_create(g)
+    g.replicaset = replicaset:new({})
+    local box_cfg = {
+        replication_timeout = 0.1,
+        replication_synchro_quorum = 2,
+        replication_synchro_timeout = 1000,
```

Comment:
too much value, isn't it?

Reply:
Why? I intentionally don't want to trigger any timeouts.